Common/src/main/java/at/petrak/hexcasting/common/casting/actions/rw/OpWrite.kt

```diff
@@ -20,17 +19,23 @@ object OpWrite : SpellAction {
     ): SpellAction.Result {
         val datum = args[0]
 
-        val (handStack, hand) = env.getHeldItemToOperateOn {
+        val (handStack) = env.getHeldItemToOperateOn {
             val datumHolder = IXplatAbstractions.INSTANCE.findDataHolder(it)
 
             datumHolder != null && datumHolder.writeIota(datum, true)
-        } ?: throw MishapBadOffhandItem.of(ItemStack.EMPTY.copy(), null, "iota.write") // TODO: hack
+        }
```

Comment:
I don't really like how duplicated this code is with OpRead, but I also don't think it's *that* big of an issue.